pkg/stanza/docs/operators/syslog_parser.md

```diff
@@ -4,18 +4,20 @@ The `syslog_parser` operator parses the string-type field selected by `parse_fro
-| `protocol`    | required         | The protocol to parse the syslog messages as. Options are `rfc3164` and `rfc5424`. |
-| `location`    | `UTC`            | The geographic location (timezone) to use when parsing the timestamp (Syslog RFC 3164 only). The available locations depend on the local IANA Time Zone database. [This page](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) contains many examples, such as `America/New_York`. |
-| `timestamp`   | `nil`            | An optional [timestamp](../types/timestamp.md) block which will parse a timestamp field before passing the entry to the output operator                                                                                               |
-| `severity`    | `nil`            | An optional [severity](../types/severity.md) block which will parse a severity field before passing the entry to the output operator                                                                                                  |
-| `if`          |                  | An [expression](../types/expression.md) that, when set, will be evaluated to determine whether this operator should be used for the given entry. This allows you to do easy conditional parsing without branching logic with routers. |
+| Field                                | Default          | Description |
+| ---                                  | ---              | ---         |
+| `id`                                 | `syslog_parser`  | A unique identifier for the operator. |
+| `output`                             | Next in pipeline | The connected operator(s) that will receive all outbound entries. |
+| `parse_from`                         | `body`           | The [field](../types/field.md) from which the value will be parsed. |
+| `parse_to`                           | `attributes`     | The [field](../types/field.md) to which the value will be parsed. |
+| `on_error`                           | `send`           | The behavior of the operator if it encounters an error. See [on_error](../types/on_error.md). |
+| `protocol`                           | required         | The protocol to parse the syslog messages as. Options are `rfc3164` and `rfc5424`. |
+| `location`                           | `UTC`            | The geographic location (timezone) to use when parsing the timestamp (Syslog RFC 3164 only). The available locations depend on the local IANA Time Zone database. [This page](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) contains many examples, such as `America/New_York`. |
+| `enable_octet_counting`              | `false`          | Wether or not to enable [RFC 6587](https://www.rfc-editor.org/rfc/rfc6587#section-3.4.1) Octet Counting on syslog parsing (Syslog RFC 5424 only).  |
+| `non_transparent_framing_trailer`    | `nil`            | The framing trailer, either `LF` or `NUL`, when using [RFC 6587](https://www.rfc-editor.org/rfc/rfc6587#section-3.4.2) Non-Transparent-Framing (Syslog RFC 5424 only). |
```

Comment:
This section also says `Some devices have also been seen to emit a two-character TRAILER, which is usually CR and LF.` Should we support that as an option?

Reply:
The library only supports `LF` and `NUL` as a [type](https://github.com/influxdata/go-syslog/blob/develop/nontransparent/trailer_type.go#L9-L16) it doesn't allow you to specify arbitrary trailers right now. So for the case of `CR and LF` if the `LF` is last it'll work still but the `CR` might get put as part of the last parsed field.

Reply:
Makes sense, thanks